packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1232,6 +1233,14 @@ export default class Meeting extends StatelessWebexPlugin {
      */
     this.keepAliveTimerId = null;
 
+    /**
+     * id for tracking Share instances in Call Analyzer
+     * @instance
+     * @type {String}
+     * @private
+     * @memberof Meeting
+     */
+    this.shareInstanceId = null;
```

Comment:
the annotation code added by Marmot team also seems to be using shareInstanceId. I'm not sure where they get it from, but it might be worth to just double check that to avoid some duplication and to make sure that we use the same shareInstanceId in our CA events as they do in annotation code.

Reply:
Looks like annotation is getting it based in from the client, which is getting it from selectShareInstanceId (from the activeMeeting). This shareinstanceId matches the one we get from contentShare.shareInstanceId.

Maybe a solution could be having transmitShareInstanceId and a remoteShareInstanceId, that way we won't have to worry about any race conditions.

Reply:
yeah, I agree, happy with localShareInstanceId and remoteShareInstanceId, or just localShareInstanceId and existing code like MediaSharesUtils.getShareInstanceId() used for the remote one